src/containers/Secrets/Secrets.js

```diff
@@ -384,6 +389,20 @@ export /* istanbul ignore next */ class Secrets extends Component {
           <>
             <h1>Secrets</h1>
             <LabelFilter {...this.props} />
+            <Tooltip
```

Comment:
Would this be more helpful on the create secret form? Maybe we want it in both places?

If we're keeping the tooltip here above the table it would be great if we could put it inline with the radio buttons instead of adding another heading here taking up vertical space. Would the icon be enough on its own?

Reply:
Yeah, I like it just on the creation page actually, that's where a user wants to create it and I like to think if you're trying to filter by a particular type you're wary of what each does, so I've moved it there

Reply:
@AlanGreene On the secret create page I've replaced the existing "legend" text with this tooltip with text, what do you think? Or I could keep the legend in and have the tooltip here but without the triggerText property? Otherwise, if there's both, we have "Type" and also "Secret type" - changes pushed up addressing the other comments too, thanks

![image](https://user-images.githubusercontent.com/12814972/76743758-0918c000-676b-11ea-870e-8664ee2a1ed9.png)